templates/serverpod_templates/projectname_server/setup-tables.bat

```diff
@@ -0,0 +1,22 @@
+@echo off
```

Comment:
This file looks good! If you make a change that only adds this file, I can approve it and fix the other stuff around it.

Reply:
Okie I will revert the code to previous one.